docs/source/use/using.rst

```diff
@@ -0,0 +1,172 @@
+=================
+Use Jupyter tools
+=================
+
+Information relevant to using the various tools in the Jupyter
+ecosystem.
+
+.. TODO: this is hidden because there's a bug in the pydata theme that won't render TOC items under headers
+
+.. toctree::
+   :hidden:
+
+   use-cases/content-user
+   ../install
+   jupyter-command
+   jupyter-directories
+   config
+   advanced/content-advanced
+
+
+Installation
+============
+
+.. toctree::
+   :maxdepth: 2
+
+   /install
+
+
+Use and Configure
+=================
+
+.. toctree::
+   :maxdepth: 1
+
+   jupyter-command
+   jupyter-directories
+   config
+
+
+How do I decide which packages I need?
+======================================
+
+Jupyter can be used for many different use-cases. Below are a few user stories
+for when and how you might use tools in the Jupyter ecosystem, as well as a diagram
+that helps lay out some of your options.
+
+.. toctree::
+
+   use-cases/content-user
+
+.. graphviz::
```

Comment:
This part doesn't seem to render ok on CircleCI doc build :confused: 

![rendering-issue](https://user-images.githubusercontent.com/7579677/87328990-01153500-c53f-11ea-8196-00b618c94bb2.png)

Reply:
I think this should render fine in readthedocs, just not circleci...I had the same problem

Reply:
Then maybe it's a good idea to enable the RTD PR build for this repo? :heart:

Reply:
I actually did and for some reason it isn't showing up, and I have no idea why haha...maybe the web hook is broken?